all.json

```diff
@@ -1,7 +1,7 @@
 {
     "id": "all",
     "summary": "Are all of the values true?",
-    "description": "Checks if **all** of the values in `data` are true. Evaluates all values from the first to the last element and stops once the outcome is unambiguous.\n\nIf only one value is given, the process evaluates to the given value. If no value is given (i.e. the array is empty) the process returns `null`.\n\nBy default all no-data values are ignored so that the process returns `null` if all values are no-data, `true` if all other values are true and `false` otherwise. Setting the `ignore_nodata` flag to `false` considers no-data values so that `null` is a valid logical object. If a component is `null`, the result will be `null` if the outcome is ambiguous. See the following truth table:\n\n```\n      || null  | false | true\n----- || ----- | ----- | -----\nnull  || null  | false | null\nfalse || false | false | false\ntrue  || null  | false | true\n```",
+    "description": "Checks if **all** of the values in `data` are true. Evaluates all values from the first to the last element and stops once the outcome is unambiguous. If no value is given (i.e. the array is empty) the process returns `null`.\n\nBy default all no-data values are ignored so that the process returns `null` if all values are no-data, `true` if all other values are true and `false` otherwise. Setting the `ignore_nodata` flag to `false` considers no-data values. If a component is `null`, the result will be `null` if the outcome is ambiguous. See the following truth table for any two consecutive values in the array:\n\n```\n      || null  | false | true\n----- || ----- | ----- | -----\nnull  || null  | false | null\nfalse || false | false | false\ntrue  || null  | false | true\n```",
```

Comment:
there is still this hard to read "if .., the result will be ... if ..." construct here

Reply:
Proposed replacement for 

>  Setting the `ignore_nodata` flag to `false` considers no-data values. If a component is `null`, the result will be `null` if the outcome is ambiguous. See the following truth table for any two consecutive values in the array:

->

>  Setting the `ignore_nodata` flag to `false` takes no-data values into account, handling them as an "ambiguous" value. The array values are reduced pairwise according to the truth table below. A `null` array item makes the result ambiguous, unless there is a `false` in the data, which makes the result unambiguously `false`.

Reply:
Thanks for the proposal. I tried to be more concise and leave it mostly up to the table to make it clear how things resolve. The "reducing" part of your proposal helps with that. Not we don't really need to explain and just let users check the table. See https://github.com/Open-EO/openeo-processes/pull/200/commits/747868f5052f3d2269a09477a2c1d9746cf32217 What do you think?